src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/Shadow/PlatformShadowEffect.ios.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.ComponentModel;
+using CoreGraphics;
+using UIKit;
+using Xamarin.CommunityToolkit.Effects;
+using Xamarin.CommunityToolkit.iOS.Effects;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.iOS;
+
+[assembly: ExportEffect(typeof(PlatformShadowEffect), nameof(ShadowEffect))]
+
+namespace Xamarin.CommunityToolkit.iOS.Effects
+{
+	public class PlatformShadowEffect : PlatformEffect
```

Comment:
The macOS Effect must be very similar to this one.

Reply:
I shall take a look:)